### DISCUSSION TITLE
Query Error: request to https://server-name failed, reason: self-signed certificate in certificate chain

### DISCUSSION THREAD

**Discussion by ATHENA12D** [Budibase/budibase#12777] [Help]
**Checklist**
- [0 ] I have searched budibase discussions and github issues to check if my issue already exists

**Hosting**

- Self
  - Method: docker single image
  - Budibase Version: 2.13.51

**Describe the bug**
Recently I developed Budibase by Docker Run method, so far everything is perfect. I created an application and created a query  to make GET request using Rest API  on this URL https://server-name , but I got this error more than once "Query Error: request to https://server-name failed, reason: self-signed certificate in certificate chain". 
My question is: How can I put the self certificate in this URL https://server-name so that the Budibase can find it when i make inquiring about this domain?

**Comment by linear** [comment]
<p><a href="https://linear.app/budibase/issue/BUDI-7895/query-error-request-to-httpsserver-name-failed-reason-self-signed">BUDI-7895 Query Error: request to https://server-name failed, reason: self-signed certificate in certificate chain</a></p>
